wazo_call_logd/tests/test_cel_interpretor.py

```diff
@@ -259,3 +261,111 @@ def test_interpret_xivo_user_fwd_regexp_with_value_after(self):
         assert_that(result.requested_name, equal_to('Bob Marley'))
         assert_that(result.requested_internal_exten, equal_to('100'))
         assert_that(result.requested_internal_context, equal_to('internal'))
+
+    def test_interpret_wazo_internal_call_has_destination_details(self):
+        cel = Mock(
+            eventtype='WAZO_CALL_LOG_DESTINATION',
+            extra='{"extra":"type: user,id: c3f297bd-93e1-46f6-a309-79b320acb7fb,name: Willy Wonka"}',
```

Comment:
In Wazo, a user can have an `id` *and* a `uuid`. In order to avoid ambiguity, it would be better to have the key `uuid` for UUIDs, instead of `id`. This also applies to `type=meeting`, but not `type=conference`